cloudhub-2/modules/ROOT/pages/ch2-deploy-api.adoc

```diff
@@ -0,0 +1,174 @@
+The `count` parameter enables you to specify the amount of ports desired. `count` can range from 1 to 10. The end result is limited by the actual availability of ports.
+
+== Deploy an App Using the Application Manager API
+
+In the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/amc-application-manager/minor/4.0/console/method/%231158/[Application Manager API], use the `tcp` section under `deploymentSettings` to map your TCP port and deploy your app. The mapping uses the external port to expose `portNumber` and the app level-defined port `applicationPortNumber`. There is a 10 port limit per application. 
+
+The following example shows the request to deploy a new app using a TCP port mapping:
+
+[source,json]
+----
+POST https://anypoint.mulesoft.com/amc/application-manager/api/v2/organizations/{organizationId}/environments/{environmentId}/deployments
+
+{
+ "name": "logs-app-uc01",
+ "labels": [
+   "beta"
+ ],
+ "target": {
+   "provider": "MC",
+   "targetId": "cfe2157f-438b-4fb8-81fe-13ec89c55545",
+   "deploymentSettings": {
+     "clustered": false,
+     "enforceDeployingReplicasAcrossNodes": false,
+//------------------------------------NEW-------------------------------------
+     "tcp": {
+         "inbound": {
+           "ports": [
+             {
+               "portNumber": 30500,
+               "applicationPortNumber": 9000  
+             },
+             {
+               "portNumber": 30501,
+               "applicationPortNumber": 9001  
+             },
+             {
+               "portNumber": 30502,
+               "applicationPortNumber": 9002  
+             },
+           ]
+         }
+       },
+//------------------------------------NEW-------------------------------------
+     "jvm": {},
+     "runtimeVersion": "4.4.0:20220523-1",
+     "updateStrategy": "rolling",
+     "disableAmLogForwarding": false,
+     "generateDefaultPublicUrl": true
+   },
+   "replicas": 1
+ },
+ "application": {
+   "ref": {
+     "groupId": "66310c16-bce5-43c4-b978-5945ed2f99c5",
+     "artifactId": "app-with-40-schedulers",
+     "version": "1.0.5",
+     "packaging": "jar"
+   },
+   "assets": [],
+   "desiredState": "STARTED",
+   "configuration": {
+     "mule.agent.application.properties.service": {
+       "applicationName": "logs-app-uc01",
+       "properties": {},
+       "secureProperties": {}
+     },
+     "mule.agent.logging.service": {
+       "artifactName": "logs-app-uc01",
+       "scopeLoggingConfigurations": []
+     }
+   },
+   "integrations": {
+     "services": {
+       "objectStoreV2": {
+         "enabled": false
+       }
+     }
+   },
+   "vCores": "0.05"
+ }
+}
+----
+
+Deployments that include unavailable ports fail. Successful deployments occupy the newly mapped ports, and those ports become unavailable. If the app is deleted, the ports become available again.
```

Comment:
PENDING: 

[vli63127](https://github.com/vli63127) [1 hour ago](https://github.com/mulesoft/docs-hosting/pull/138/files#r1253528586)
add what the error message will be shown... does it specifically mention it's due to port unavailability?

Member
@[afelisatti](https://github.com/afelisatti) afelisatti [45 minutes ago](https://github.com/mulesoft/docs-hosting/pull/138/files#r1253605541)
It should AFAIK, @rsaxena05 can you confirm the exact message?

@rsaxena05

Reply:
```suggestion
Deployments that include unavailable ports fail. Successful deployments occupy the newly mapped ports, and those ports become unavailable. If the app is deleted, the ports become available again.

If you configure ports incorrectly, the following errors appear:

* When you specify more than ten ports: `Owner <name>  has reached the maximum port lease capacity`
* When the port you specify is not in the range of 30500 to 32500: `Ports must be within the supported range of 30500 - 32500`
* When the port you specify is in use by another app: `Ports 30501 is already in use by other owner`
```